python_modules/dagster/dagster/_core/definitions/freshness_evaluator.py

```diff
@@ -48,7 +48,7 @@ async def evaluate_freshness(
         If the asset was materialized within the policy time window but outside the warning time window (if provided), return WARN.
         """
         freshness_policy = check.not_none(
-            InternalFreshnessPolicy.from_asset_spec_metadata(node.metadata)
+            node.freshness_policy or InternalFreshnessPolicy.from_asset_spec_metadata(node.metadata)
```

Comment:
it seems easy to accidentally mess this up in a new callsite and forget the old codepath, should it be a field on BaseAssetNode?